docs/src/RunningClimateMachine.md

```diff
@@ -0,0 +1,121 @@
+                        [--monitor-courant-numbers <interval>]
+                        [--checkpoint <interval>]
+                        [--checkpoint-keep-all] [--checkpoint-at-end]
+                        [--checkpoint-dir <path>]
+                        [--restart-from-num <number>]
+                        [--log-level <level>] [--output-dir <path>]
+                        [--integration-testing] [-h]
+
+Climate Machine: an Earth System Model that automatically learns from data
+
+optional arguments:
+  -h, --help            show this help message and exit
+
+ClimateMachine:
+  --disable-gpu         do not use the GPU
+  --show-updates <interval>
+                        interval at which to show simulation updates
+                        (default: "60secs")
+  --diagnostics <interval>
+                        interval at which to collect diagnostics
+                        (default: "never")
+  --vtk <interval>      interval at which to output VTK (default:
+                        "never")
+  --monitor-timestep-duration <interval>
+                        interval in time-steps at which to output
+                        wall-clock time per time-step (default:
+                        "never")
+  --monitor-courant-numbers <interval>
+                        interval at which to output acoustic,
+                        advective, and diffusive Courant numbers
+                        (default: "never")
+  --checkpoint <interval>
+                        interval at which to create a checkpoint
+                        (default: "never")
+  --checkpoint-keep-all
+                        keep all checkpoints (instead of just the most
+                        recent)
+  --checkpoint-at-end   create a checkpoint at the end of the
+                        simulation
+  --checkpoint-dir <path>
+                        the directory in which to store checkpoints
+                        (default: "checkpoint")
+  --restart-from-num <number>
+                        checkpoint number from which to restart (in
+                        <checkpoint-dir>) (type: Int64, default: -1)
+  --log-level <level>   set the log level to one of
+                        debug/info/warn/error (default: "info")
+  --output-dir <path>   directory for output data (default: "output")
+  --integration-testing
+                        enable integration testing
+
+Any <interval> unless otherwise stated may be specified as:
+    - 2hours or 10mins or 30secs => wall-clock time
+    - 9.5smonths or 3.3sdays or 1.5shours => simulation time
+    - 1000steps => simulation steps
+    - never => disable
+    - default => use experiment specified interval (only for diagnostics at
+      present)
+```
+
+There may also be driver-specific command line arguments.
+
+## Environment variables
+
+Currently, the `ClimateMachine` only looks at one environment variable --
+`CLIMATEMACHINE_GPU`, which if set to `false`, disables GPU use.
```

Comment:
Aren't there other environmental variables available, like `TICTOC_PRINT_RESULTS=1`, or were these rolled into the command-line arguments?

Reply:
That one is mainly for internal inspection right now. It would perhaps be described in `DevDocs` somewhere at some point...